bu-ssl.php

```diff
@@ -338,31 +353,19 @@ public function update_post( $post_id ) {
 		$urls = self::search_for_insecure_content( $post->post_content );
 
 		// Update post meta with list of insecure urls.
-		self::do_update_postmeta( $this->options['post_meta_key'], $post_id, $urls );
+		update_post_meta( $post_id, $this->options['post_meta_key'], $urls );
```

Comment:
Great change, especially as d`o_update_postmeta` swaps the param order for `update_post_meta` which is real confusing!